tests/pytest.ini

```diff
@@ -1,12 +1,13 @@
 [pytest]
-addopts = -ra --module-path=../ansible/library/ --show-capture=no --ignore=ptftests --ignore=acstests --ignore=saitests --ignore=test_vrf.py
 markers:
     acl: ACL tests
     bsl: BSL tests
     reboot: tests which perform SONiC reboot
     port_toggle: tests which toggle ports
     disable_loganalyzer: make to disable automatic loganalyzer
     broadcom: test specific to Broadcom platform
+    pretest: tests are run before feature/regression test cases to prepare the DUT/environment.
+    posttest: tests are run after feature/regression test cases to cleanup the DUT/environment and/or collect diagnostics.
```

Comment:
Which test uses this marker? is it a placeholder?

Reply:
Place holder for now. the cleanup tests will go to this group.